test/runtime/Policies.go

```diff
@@ -259,6 +259,11 @@ var _ = Describe("RuntimePolicies", func() {
 
 	BeforeAll(func() {
 		vm = helpers.InitRuntimeHelper(helpers.Runtime, logger)
+
+		// Make sure that Cilium is started with appropriate CLI options
+		// (specifically to exclude the local addresses that are populated for
+		// CIDR policy tests).
+		Expect(vm.SetUpCilium()).To(BeNil())
```

Comment:
This and the following change seem independent from this PR, and may need to be backported to older releases? As such maybe split these to a separate PR?

Reply:
Created https://github.com/cilium/cilium/pull/8548